enhancements/ingress/optional-ingress.md

```diff
@@ -0,0 +1,225 @@
+  - Make the cluster authentication operator tolerate the absence of the default ingress controller.
+  - Make the cluster monitoring operator tolerate the absence of the default ingress controller.
+- Disable the Route API or try to tolerate the absence of the default ingress controller.
+- Implement the ingress operator disabling in the HyperShift's control plane operator.
+
+## Proposal
+
+### User Stories
+
+- As a HyperShift engineer, I want to be able to disable the ingress operator on the hosted control plane.
+- As an OpenShift engineer, I want the cluster operators used by HyperShift to tolerate the absence of the ingress operator.
+- As a HyperShift hosted cluster's user, I want to see the ingress capability as a known capability on my cluster.
+
+### Workflow Description
+
+The HyperShift hosted cluster's user wants to use an AWS ALB to route the traffic to its example application deployed on a HyperShift's hosted cluster.
+
+#### Terminology
+[HyperShift concepts and personas](https://hypershift-docs.netlify.app/reference/concepts-and-personas/).
+
+#### Disable the default ingress
+
+- Cluster service consumer requests a hosted cluster without the ingress capability using the HostedCluster API (**note**: the HostedCluster's capability API does not exist and it's out of scope of this enhancement).
+- HyperShift creates the hosted cluster.
+- HyperShift skips the deployment of the ingress operator on the hosted control plane.
+- HyperShift creates the ingress operator's `ClusterVersion` without the ingress capability on the hosted cluster.
+- Hosted cluster's cluster version operator does not deploy the cluster ingress operator's payload (CRDs, RBACs, etc.).
+
+#### Setup AWS ALB as the new ingress
+
+- Cluster instance admin installs [AWS Load Balancer Operator](/enhancements/ingress/aws-load-balancer-operator.md).
+- Cluster instance admin sets up the default `AWSLoadBalancerController`.
+- Cluster instance user creates the `Ingress` resource for the default aws load balancer controller's ingress class.
+- AWS load balancer controller provisions an ALB on AWS.
+- Cluster instance admin can install [External DNS Operator](/enhancements/dns/externaldns-operator.md) to help cluster instance users set up convenient DNS names.
+
+### API Extensions
+
+N/A - no api extensions are being introduced in this proposal
+
+### Implementation Details/Notes/Constraints
+
+The name of the new capability is `Ingress`. It includes the cluster ingress operator and all its APIs listed below.
+The implementation of the new capability follows the instructions of [how to implement a new capability](/enhancements/installer/component-selection.md#how-to-implement-a-new-capability).
+
+#### HyperShift
+
+HyperShift doesn't pass all the ingress operator's payload to the cluster version operator.
+Certain manifests are filtered out, most importantly: the ingress operator's deployment.
```

Comment:
if it's already being filtered out, what changes with the introduction of this capability?

Reply:
It's detailed below but mainly the rest of the payload (IngressController CRD, DNSRecord CRD, RBACs, etc.). It won't be applied anymore.